src/website/app/Router.js

```diff
@@ -70,7 +70,11 @@ export default function Router({ demos }: Props) {
                 <IconArrowBack color="currentColor" size="small" />{' '}
                 {selectedDemo.title}
               </Link>
-              <ComponentDocExample {...selectedExample} />
+              <ComponentDocExample
+                chromeless
```

Comment:
I can't really tell what you are trying to do here with the `chromeless` prop.  This is setting `chromeless` to true for all `ComponentDocExample` child pages even when they are not `chromeless`, which I don't think is what we want.

e.g. http://0.0.0.0:8080/components/button/primary/